go/service/kbfs_mount.go

```diff
@@ -62,7 +62,7 @@ func (h *KBFSMountHandler) WaitForMounts(ctx context.Context) (active bool, err
 			// Not check os.IsNotExist here because it can be permission
 			// error too. So just wait it out.
 		case <-ctx.Done():
-			return false, ctx.Err()
+			return false, nil
```

Comment:
@strib I changed this to just return `nil` instead of the timeout error. In GUI we don't really use the returned boolean, but instead just refresh driver status afterwards which should cover both cases. Not returning an error here allows us to not putting that call in a `try` block to cover the timeout case. Please let me know if this doesn't sound good to you!